pkgs/top-level/all-packages.nix

```diff
@@ -20159,7 +20157,7 @@ with pkgs;
 
   qimageblitz = callPackage ../development/libraries/qimageblitz {};
 
-  qjson = callPackage ../development/libraries/qjson { };
+  qjson = libsForQt5.callPackage ../development/libraries/qjson { };
```

Comment:
Libraries must be built for all versions of Qt 5. Please move this to `qt5-packages.nix` (or update your pull request to allow maintainers to edit it).